test/living-html/htmlinputelement.js

```diff
@@ -148,3 +149,30 @@ exports["an input's parsed type attribute should be reflected in both its proper
 
   t.done();
 };
+
+exports["cloning a radio input also clones its dirty checkedness"] = t => {
+  const doc = jsdom.jsdom();
+  const input = doc.createElement("input");
+  input.setAttribute("type", "radio");
+  input.checked = true;
+
+  const copy = input.cloneNode();
+  const impl = idlUtils.implForWrapper(copy);
+
+  t.equal(impl._dirtyCheckedness, true);
+
+  t.done();
+};
+
+exports["cloning an input also clones its dirty value"] = t => {
+  const doc = jsdom.jsdom();
+  const input = doc.createElement("input");
+  input.value = "foo bar";
+
+  const copy = input.cloneNode();
+  const impl = idlUtils.implForWrapper(copy);
+
+  t.equal(impl._dirtyValue, true);
```

Comment:
The section in the spec which describes the modified behaviour is [here](https://html.spec.whatwg.org/#attr-input-checked) and [here](https://html.spec.whatwg.org/#attr-input-value) btw.